helm/templates/deployer/deployerServiceAccount.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if eq .Release.Namespace .Values.deployer.namespace }}
+
+kind: ServiceAccount
```

Comment:
Tech debt: I don't see where grabbing the secret token corresponding to this new ServiceAccount is documented? We know how to do this now, but future us might not remember.